tasks-dev/assemble.js

```diff
@@ -3,9 +3,8 @@ const assemble = require('assemble');
 const helpers = require('handlebars-helpers');
 const debug = require('gulp-debug');
 const plumber = require('gulp-plumber');
-const newer = require('gulp-newer');
+// const newer = require('gulp-newer');
```

Comment:
Would be good to keep it in for now – might be able to fix it at some point and make it work somehow…